kqueen/blueprints/api/api3_0.yml

```diff
@@ -260,26 +416,170 @@ components:
+          properties:
+            service_account_info:
+              description: "Google cloud account json file
+                            https://cloud.google.com/video-intelligence/docs/common/auth"
+              type: "object"
+              properties:
+                auth_provider_x509_cert_url:
+                  type: "string"
+                auth_uri:
+                  type: "string"
+                client_email:
+                  type: "string"
+                client_id:
+                  type: "string"
+                client_x509_cert_url:
+                  type: "string"
+                private_key:
+                  type: "string"
+                private_key_id:
+                  type: "string"
+                project_id:
+                  type: "string"
+                token_uri:
+                  type: "string"
+        created_at:
+          type: "string"
+          format: "date-time"
+        owner:
+          type: "string"
+          description: "Reference to the related User"
+          example: "User:<uuid>"
+      required:
+        - "name"
+        - "engine"
+        - "owner"
+    AzureProvisioner:
+      type: "object"
+      description: "Set up with UI: http://kqueen.readthedocs.io/en/latest/kqueen.html#provision-a-kubernetes-cluster-using-azure-kubernetes-service
+                    Set up with CLI: http://kqueen.readthedocs.io/en/latest/kqueen.html#azure-cli-operations-with-kqueen"
+
+      properties:
+        id:
+          type: "string"
+          format: "uuid"
+        name:
+          type: "string"
+        engine:
+          type: "string"
+          enum:
+            - "kqueen.engines.AksEngine"
+        state:
+          type: "string"
+          enum:
+            - "OK"
+            - "Error"
+            - "Not Reachable"
+        parameters:
+          description: "Azure-specific parameters"
           type: "object"
+          properties:
+            client_id:
+              type: "string"
+              description: "Application ID"
+            secret:
+              type: "string"
+              description: "secret keyL (add link)"
+            tenant:
+              type: "string"
+              description: "Directory ID"
+            subscription_id:
+              type: "string"
+            resource_group_name:
+              type: "string"
         created_at:
           type: "string"
           format: "date-time"
         owner:
           type: "string"
-          description: "Reference to related User"
+          description: "Reference to the related User"
+          example: "User:<uuid>"
+      required:
+        - "name"
+        - "engine"
+        - "owner"
+    JenkinsProvisioner:
+      type: "object"
+      properties:
+        id:
+          type: "string"
+          format: "uuid"
+        name:
+          type: "string"
+        engine:
+          type: "string"
+          enum:
+            - "kqueen.engines.JenkinsEngine"
+        state:
```

Comment:
we don't have refs to state like in cluster case?

Reply:
Good point. Added provisioner state. Actually values can changed, since it is defined in config, but I don't think that smdy will change them